EventEmitter.d.ts

```diff
@@ -19,30 +19,30 @@ declare namespace EventEmitter {
         [event: string]: Listener[];
     }
 
-    type Listeners = Listener[]|ListenerMap;
+    type Listeners = Listener[] | ListenerMap;
 
-    export class EventEmitter {
+    export class EventEmitter<E = EventKey> {
```

Comment:
So `<E = EventKey>` turns `EventEmitter` into a generic class with a type variable that defaults to `EventKey` if not provided? (I don't know TS, so I'm speculating! :smile:)